markdown/en/api/ff-searchbox.api.md

```diff
@@ -41,7 +41,7 @@ ___
 | **add-params**&nbsp;(String) (default: empty)| _The parameter string has to be URL-encoded._ With this property you can deliver standard parameters which will then be attached to the search request. Example: add-params="param1=abcd,param2=xyz"|
 | **add-tracking-params**&nbsp;(String) (default: empty)| With this property you can deliver standard parameters which are attached to every tracking request. Example: add-tracking-params="param1=abcd,param2=xyz"|
 | **keep-filters**&nbsp;(Boolean) **Options**: &nbsp;true, &nbsp;false (default: false)| With this property you can determine, if filters which where set before the search (e.g. via ASN) should be kept or discarded.|
-| **keep-url-params**&nbsp;(String) (default: empty)| Comma separated values; specifies the url-parameters that should be included in requests to FACT-Finder. *This is particularly useful if you only want to keep your own custom non-FF attributes in the URL.* If this is set to `all`, every parameter will be used |
+| **keep-url-params**&nbsp;(String) (default: empty)| Comma separated values; specifies the url-parameters that should be included in requests to FACT-Finder. If this is set to `all`, every parameter will be used. *This is particularly useful if you want to keep your own custom attributes in the URL whilst hiding the FACT-Finder url parameters with* `use-paramter="false"` |
```

Comment:
Sounds nice!
Looks like there is a type in `use-paramter`.
Where does that attribute actually come from? There is a `use-url-parameter` documented.
While at it, there is another typo "_serach_" in the document. Maybe you could fix that, too.

When we are talking about _parameters_ here, is `if you want to keep your own custom attributes in the URL` actually correct? Should it be `parameters` here as well?